src/promptflow/promptflow/_core/tool_meta_generator.py

```diff
@@ -221,6 +232,21 @@ def collect_tool_function_in_module(m):
     return tools[0]
 
 
+def collect_script_tools(m):
```

Comment:
Why don't we use this function to replace collect_tool_function_in_module completely?